Sources/TUSKit/TUSClient.swift

```diff
@@ -332,12 +332,15 @@ public final class TUSClient {
     /// Retry a failed upload. Note that `TUSClient` already has an internal retry mechanic before it reports an upload as failure.
     /// If however, you like to retry an upload at a later stage, you can use this method to trigger the upload again.
     /// - Parameter id: The id of an upload. Received when starting an upload, or via the `TUSClientDelegate`.
-    /// - Returns: True if the id is found. False if it's not found
+    /// - Returns: False if the id is found. True if it's not found
```

Comment:
The documentation for return could be improved I think. Looking at the code it looks like it'll return False if the id is already in the `uploads` property OR the upload metadata can't be found via `files.findMetadata()`. The current documentation is not accurate and could be confusing to a user.

Reply:
Maybe something like:
```swift
/// - Returns: True if the upload has been scheduled for a new upload attempt. False if the upload id was not found or does not correspond to a failed upload.
```
I think that would better describe what's being returned.

Reply:
Agreed! I'll take a look but your wording seems good. Thanks for this input